input.go

```diff
@@ -23,6 +23,28 @@ func MouseEvent(typ input.MouseType, x, y float64, opts ...MouseOption) MouseAct
 	return p
 }
 
+
+
+
+
+func MouseMoveXY(x, y float64, opts ...MouseOption) MouseAction {
+	return ActionFunc(func(ctx context.Context) error {
+		p := &input.DispatchMouseEventParams{
+			Type:       input.MouseMoved,
+			X:          x,
+			Y:          y,
+		}
```

Comment:
better to use the constructor func?

```go
p := input.DispatchMouseEvent(input.MouseMoved, x, y)
```

Reply:
Yes, I just noticed this function here, it did the thing what I’m looking for for MouseMoveXY